cpp/ql/src/Security/CWE/CWE-416/UseOfStringAfterLifetimeEndsGood.cpp

```diff
@@ -0,0 +1,7 @@
+#include <string>
+void work(const char*);
+
```

Comment:
Similarly, we should add a `GOOD` comment here.

Reply:
Thanks! Added in 351caaccfe59ae1ba7d5bc73fdf9b90c38d3b6e5.